lib/MusicBrainz/Script/RemoveEmpty.pm

```diff
@@ -122,6 +122,16 @@ sub run {
                     editor => $modbot,
                     privileges => $BOT_FLAG | $AUTO_EDITOR_FLAG
                 );
+
+                $self->c->model('EditNote')->add_note(
+                    $edit->id,
+                    {
+                        editor_id => $EDITOR_MODBOT,
+                        text => localized_note('This entity was automatically removed because it was empty: it had no relationships associated with it, nor (if relevant for the type of entity in question) any recordings, releases nor release groups.
+                                 If you consider this was a valid, non-duplicate entry that does belong in MusicBrainz, feel free to add it again, but please ensure enough data is added to it this time to avoid another automatic removal.')
```

Comment:
Oops, meant to request changes. You also have to wrap the string with `N_l` here